src/components/ui/shadcn-io/dropzone/index.tsx

```
@@ -0,0 +1,202 @@
+    unitIndex++;
+  }
+
+  return `${size.toFixed(2)}${units[unitIndex]}`;
+};
+
+const DropzoneContext = createContext<DropzoneContextType | undefined>(
+  undefined
+);
+
+export type DropzoneProps = Omit<DropzoneOptions, 'onDrop'> & {
+  src?: File[];
```

ESLint

[error] Delete `;`

(prettier/prettier)